scripts/roads/ingest_roads_rest.py

```
@@ -173,11 +175,20 @@ def main():
-        sys.exit()
-    jobj = req.json()
+    LOG.info("Found %s current entries", len(current))
+    for attempt in range(2):
+        LOG.info("Attempt %s/1 fetching", attempt)
+        try:
+            resp = requests.get(URI, timeout=30)
+            LOG.info("Got status_code %s", resp.status_code)
+            resp.raise_for_status()
+            jobj = resp.json()
+            break
+        except Exception as exp:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)